src/main/webapp/script.js

```diff
@@ -11,3 +11,73 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//
+
+// Does Post request to get sentiment score
+async function postSentiment(data) {
+  // TODO: We want to get our message from our translation servlet in the future
+
+  // POST Request
+  const response = await fetch("/sentiment", {
+    method: "POST", // Send Post Request to /sentence
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(data),
+  });
+
+  // Sentiment value from POST response
+  const sentiment = await response.text();
+
+  return sentiment;
+}
+
+// Gets a sentiment value and updates DOM with new element.
+async function getSentiment() {
+  // Depends on where our value is stored in our index.html
+  // TODO: This value should be updated to get our message from our translation servlet.
+  const data = document.getElementById("message").value;
+
+  // Gets our sentiment vaue from POST Request
+  const sentiment = await postSentiment(data);
+
+  // Depends on the element id used to display our sentiment
+  const sentimentContainer = document.getElementById("sentiment");
+  sentimentContainer.innerText = "";
+
+  sentimentContainer.appendChild(
+    createParagraphElement("Our sentiment score: " + sentiment)
+  );
+
+  sentimentContainer.appendChild(
+    createParagraphElement(getMessage(parseFloat(sentiment)))
+  );
+}
+
+// Creates a <p> element containing text.
+function createParagraphElement(text) {
+  const pElement = document.createElement("p");
+  pElement.innerText = text;
+  return pElement;
+}
+
+// This gets a message value based on sentiment scores
+function getMessage(score) {
```

Comment:
Where is this function used?

Reply:
This is used in the getSentiment funciton.

Reply:
Got it. thanks!